app/views/transactionsRerunModal.html

```diff
@@ -28,7 +28,25 @@ <h4 ng-show="transactionsSelected.length > 1">Your selected transactions have be
 
   </div>
   <div class="modal-footer">
+    <div ng-show="transactionsSelected.length > 1" ng-if="rerunSuccess != true" style="float: left;">
+      <label style="float: left; margin: 10px 10px 0px 0px;" tooltip="The number of transactions that the task processor should run concurrently">Batch size</label>
+      <select style="float: left; width: auto;" class="form-control" ng-model="taskSetup.batchSize">
+        <option value="1">One at a time</option>
+        <option ng-if="transactionsSelected.length >= 2" value="2">2</option>
```

Comment:
Minor: I wonder if we should make these all append 'at a time' eg. 2 at a time etc.